### PR TITLE
Use image magick stream in resize photos

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

File: server/lib/thumb.coffee [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fs = require 'fs'
-gm = require 'gm'
+gm = require('gm').subClass({ imageMagick: true })
```
Comment (frankrousseau) [inline server/lib/thumb.coffee]: Braces are useless.

**Comment by frankrousseau** [comment]
Thumb tests are broken.

**Comment by poupotte** [comment]
@aenario, @frankrousseau , can you review ?
I tested this patch on my Cozy production with 10 - 20 - 50 -150 files, it works but it take time to generate all thumbnails : about 10 by minute, so 15' for 150 files. However, I think it's better to have a long execution for the moment.

**Comment by frankrousseau** [comment]
Do you think it will work in case where the user upload 2000 photos from his phone?

**Comment by frankrousseau** [comment]
@poupotte Did you see my question?

**Comment by poupotte** [comment]
Sorry for the delay. I think it's work but it takes long time between pictures synchronisation and thumb creation because we use a queue to create thumb. However, I can test to be sure.

Anyway, this PR should work better than actual master. Indeed, the difference is more stream use and less information stored in RAM (only id instead of total file document)

**Comment by clochix** [comment]
This PR is opened for quite a long time now, @frankrousseau  could we merge it and thank @poupotte for her awesome work ?

**Comment by frankrousseau** [comment]
Ok. @poupotte can you rebase it? Then you can merge it by yourself.

**Pull request merged**